doc/apimanager-reference.md

```diff
@@ -106,6 +107,10 @@ Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdow
 | Resources | `resources` | [v1.ResourceRequirements](https://v1-17.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#resourcerequirements-v1-core) | No | `nil` | Resources describes the compute resource requirements. Takes precedence over `spec.resourceRequirementsEnabled` with replace behavior |
 | Workers | `workers` | integer | No | Automatically computed. Check [apicast doc](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#apicast_workers) for further info. | Defines the number of worker processes |
 | LogLevel | `logLevel` | string | No | N/A | Log level for the OpenResty logs  (see [docs](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#apicast_log_level)) |
+| CustomPolicies | `customPolicies` | [][CustomPolicySpec](#CustomPolicySpec) | No | N/A | List of custom policies |
+| HTTPSPort | `httpsPort` | int | No | **8443** only when `httpsCertificateSecretRef` is provided | Controls on which port APIcast should start listening for HTTPS connections. Do not use `8080` as HTTPS port (see [docs](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#apicast_https_port)) |
```

Comment:
Have we checked if there is some other component that is referencing the APIcast port or doing some assumption about its value?

Reply:
Apicast is not, as we are using the same in the apicast operator and it is working.

It does not have any sidecar container in the pod, so far. Thus it should be available.

Reply:
Yes, I understand that.
My question is if some other component relies on the communication with apicast through http and we would need to change that too. For example, system to apicast communication

Reply:
the HTTP port (called `gateway` in the service) is still enabled. So other components should not be affected. Unless there is a conflict, in which case only TLS will be served [link](https://github.com/3scale/APIcast/blob/master/doc/parameters.md#apicast_https_port).

The doc explicitly states `Do not use 8080 as HTTPS port` to avoid conflicts.

Reply:
I see. Got you. I still have some doubts about this though.
Ok, we have the 'gateway' port and the 'httpsproxy' port.
When we enable TLS the httpsproxy port is created and used in both the APIcast pod and APIcast service, which means that someone could use the Service to access APIcast through that pot in the `Service`.
However, the components inter-communication would still be through HTTP and not through HTTPS right? is that the intended behavior that we want? If that's the intended behavior I think we should clarify this in the documentation because a user might think/interpret that when they set TLS on APIcast that means that automatically all communication to it would be encrypted, which would not be the case.

Reply:
Ok, The operator does not provide a way to "disable" the HTTP when TLS is enabled. That is wanted?

Reply:
Added doc  to make this point clear

Reply:
https://github.com/3scale/3scale-operator/pull/632/files?file-filters%5B%5D=.go&file-filters%5B%5D=.md#diff-72dab7f3126dc7dbd85ab6398f36d48315960176cddc220c7fd9bdb36da97fb6R6

Reply:
:+1: